src/canvas/view/FrameView.ts

```diff
@@ -305,18 +321,18 @@ export default class FrameView extends View {
     find(prevStyles, styles, toRemove);
     toRemove.forEach(stl => {
       const el = head.querySelector(`link[href="${stl.attributes.href}"]`);
-      el && el.parentNode.removeChild(el);
+      el?.parentNode?.removeChild(el);
     });
     appendVNodes(head, toAdd);
   }
 
   renderBody() {
-    const { config, model, ppfx } = this;
-    const { em } = config;
+    const { config, em, model, ppfx } = this;
     const doc = this.getDoc();
     const body = this.getBody();
     const win = this.getWindow();
-    const conf = em.get('Config');
+    const conf = em.config;
+    //@ts-ignore TODO I don't understand why this needed nowhere else is used
```

Comment:
The idea was to make use of it inside custom components with scripts so you could change the behavior of components during the editing (eg. disable JS animations if the element is in the editor) but in all honesty, I'm not sure anymore if it's the right approach. For now, let's leave it here.